apps/web/src/components/LatteLayout/index.tsx

```diff
@@ -9,7 +12,11 @@ export function LatteLayout({ children }: { children: React.ReactNode }) {
       initialPercentage={50}
       minSize={300}
       firstPane={children}
-      secondPane={<LatteChat />}
+      secondPane={
+        <ClientOnly>
+          <LatteChat />
```

Comment:
nextjs prerenders this component server-side but not children in some cases, which results in a juttery UX, forcing client side on this one.